target/linux/mpc85xx/patches-4.14/104-powerpc-mpc85xx-change-P2020RDB-dts-file-for-OpenWRT.patch

```diff
@@ -0,0 +1,128 @@
+From 3465eb7c461a2d3cf4326352a139db43fbcc15fd Mon Sep 17 00:00:00 2001
+From: Pawel Dembicki <paweldembicki@gmail.com>
+Date: Sun, 30 Dec 2018 23:24:41 +0100
+Subject: [PATCH] powerpc: mpc85xx: change P2020RDB dts file for OpenWRT
+
+This patch apply chages for OpenWRT in P2020RDB
+dts file.
+
+Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
+---
+ arch/powerpc/boot/dts/fsl/p2020rdb.dts | 43 +++++++++++++++++++++-----
+ 1 file changed, 35 insertions(+), 8 deletions(-)
+
+diff --git a/arch/powerpc/boot/dts/fsl/p2020rdb.dts b/arch/powerpc/boot/dts/fsl/p2020rdb.dts
+index 435a319958cb..41320958ce98 100644
+--- a/arch/powerpc/boot/dts/fsl/p2020rdb.dts
++++ b/arch/powerpc/boot/dts/fsl/p2020rdb.dts
+@@ -2,6 +2,7 @@
+  * P2020 RDB Device Tree Source
+  *
+  * Copyright 2009-2012 Freescale Semiconductor Inc.
++ * Copyright 2018      Pawel Dembicki <paweldembicki@gmail.com>
+  *
+  * This program is free software; you can redistribute  it and/or modify it
+  * under  the terms of  the GNU General  Public License as published by the
+@@ -9,10 +10,15 @@
+  * option) any later version.
+  */
+ 
++/dts-v1/;
++
++#include <dt-bindings/gpio/gpio.h>
++#include <dt-bindings/input/input.h>
++
+ /include/ "p2020si-pre.dtsi"
+ 
+ / {
+-	model = "fsl,P2020RDB";
++	model = "Freescale P2020RDB";
```

Comment:
This is complicated. The current [DT Spec](https://github.com/devicetree-org/devicetree-specification/blob/211bbc01adb5bc5e64cb651651c4b907ed2d309f/source/devicetree-basics.rst#model) 



    Description:

        The model property value is a <string> that specifies the manufacturer’s model number of the device.

        The recommended format is: "manufacturer,model", where manufacturer is a string describing the name of the manufacturer (such as a stock ticker symbol), and model specifies the model number.

    Example:

        model = "fsl,MPC8349EMITX";

so :laughing: . 

Note:
I'm totally fine with "Freescale P2020RDB". This is just a FYI in case you want to upstream some of your changes.

Reply:
Thank You for information.